src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -913,9 +913,36 @@ http://www.postgresql.org[Postgres] or
 http://www.microsoft.com/en-us/server-cloud/products/sql-server/[SQL Server] if
 you care about performance or notice a slowdown.
 
-ProActive Workflows and Scheduling distribution is provided with several samples
-for configuring the Scheduler and Resource Manager to use most standard RDBMS.
-You can find these examples in `PROACTIVE_HOME/config/scheduler/` folder.
+ProActive Workflows and Scheduling distribution is provided with several
+samples for configuring the Scheduler and Resource Manager to use most
+standard RDBMS.
+You can find these examples in `PROACTIVE_HOME/config/rm/templates/` and
```

Comment:
The documentation references the changes introduced by ow2-proactive/scheduling#2537. Except if you suggest another folder for database templates, the paths should be correct.